src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml

```diff
@@ -31,6 +32,14 @@
         <div data-test-id="login-page">
             <div class="nhsuk-grid-row">
                 <div class="nhsuk-grid-column-full">
+                    <div data-test-id="go-back-section" class="nhsuk-back-link">
+                        <a data-test-id="go-back-link" class="nhsuk-back-link__link" href="/">
```

Comment:
will this href work?

Reply:
It currently does, locally it reroutes to "localhost:8070/", we've done a similar thing in the account controller HttpGet Login method if the return url is null, which redirects them to the same route